components/script/dom/globalscope.rs

```diff
@@ -167,6 +172,14 @@ impl GlobalScope {
         }
     }
 
+    pub fn structured_clone_holder(&self) -> Option<RefMut<Vec<DomRoot<Blob>>>> {
+        if let Ok(holder) = self.structured_clone_holder.try_borrow_mut() {
```

Comment:
I don't think we want to silently not perform a structured clone if we have borrowing conflicts. We should loudly proclaim the conflicts by calling borrow_mut instead, and not return an Option value from this method.